cmd/follow.go

```diff
@@ -14,31 +14,36 @@ type options struct {
 	cfg *settings.Config
 }
 
+//followProject gets the remote data and attempts to follow its git project
 func followProject(opts options) error {
 
 	remote, err := git.InferProjectFromGitRemotes()
-
 	if err != nil {
 		return errors.Wrap(err, errorMessage)
 	}
 
-	vcsShort := "gh"
-	if remote.VcsType == "BITBUCKET" {
-		vcsShort = "bb"
-	}
-	res, err := api.FollowProject(*opts.cfg, vcsShort, remote.Organization, remote.Project)
-	if err != nil {
-		return err
-	}
-	if res.Followed {
-		fmt.Println("Project successfully followed!")
-	} else if res.Message == "Project not found" {
-		fmt.Println("Unable to determine project slug for CircleCI (slug is case sensitive).")
-	}
+	//check that project url contains github or bitbucket; our legacy vcs
+	if remote.VcsType == git.GitHub || remote.VcsType == git.Bitbucket {
```

Comment:
I noticed there's similar logic in `cmd/open.go`. Would it make sense to add an `isLegacyVcs` method to the `Remote` struct in `git/git.go`?

Reply:
That also would definitely work. While I was writing this I was under the assumption that it was part of an external package. That being said, I dont know if its worth going back and changing, since these are most likely going to be the only two places using this and we're looking to eventually get rid of VCSType completely.